tba_dev_config.json

```diff
@@ -1,4 +1,5 @@
 {
+    "auth_use_emulator": true,
```

Comment:
- [ ] As mentioned above - let's rename this something like `auth_use_prod` and have this default to true